app/views/events/hbx_enrollment/_policy.xml.haml

```diff
@@ -20,12 +20,14 @@
             %dba= employer.dba
           %fein= employer.fein
         %total_employer_responsible_amount=(hbx_enrollment.total_employer_contribution).round(2)
+        %composite_rating_tier_name= "urn:openhbx:terms:v1:composite_rating_tier##{hbx_enrollment.enrollment_composite_rating_tier}" if hbx_enrollment.benefit_group.plan_option_kind == "single_plan" # for only composite rated benefit group, if benefit group has single plan then it is composite rated benefit group
     - else
       %individual_market
         %is_carrier_to_bill true
         %applied_aptc_amount= (hbx_enrollment.applied_aptc_amount).round(2)
     %premium_total_amount= hbx_enrollment.total_premium.round(2)
     %total_responsible_amount= (hbx_enrollment.is_shop? ? (hbx_enrollment.total_employee_cost) : (hbx_enrollment.total_premium - hbx_enrollment.applied_aptc_amount.to_f)).round(2)
+    %rating_area= hbx_enrollment.employer_profile.rating_region
```

Comment:
Only include rating area if it has a value, otherwise leave the tag off completely.